src/Core/Helpers/StringHelper.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * String Helpers
+ *
+ * @copyright 2020 Wayfair LLC - All rights reserved
+ */
+
+namespace Wayfair\Helpers;
+
+class StringHelper {
+
+  /**
+   * Function that hides characters in a string, will default to the first and last characters being visible.
+   *
+   * @param string $string The to string to be masked
+   * @param mixed $start The position to start masking, default to position 1
+   * @param mixed $finish The position to stop masking, default to position -1
+   *
+   * @return string
+   */
+  public static function maskString($string, $start = 1, $finish = -1 ) {
```

Comment:
Do you think that `$start and $finish` should be const?

Reply:
No, but you'll need to do bounds checking on these inputs to avoid exceptions from `substr` and `substr_replace`